app/medInria/medFilteringWorkspace.cpp

```diff
@@ -44,6 +44,7 @@ medFilteringWorkspace::medFilteringWorkspace(QWidget *parent): medAbstractWorksp
 {
     d->filteringToolBox = new medFilteringSelectorToolBox(parent);
     connect(d->filteringToolBox,SIGNAL(processFinished()),this,SLOT(onProcessSuccess()));
+    d->filteringToolBox->setWorkspace(this);
```

Comment:
As said on my last comment of this PR:

> workspace set in medFilteringWorkspace

This means we need sometimes to have access to the workspace from a medFilteringWorkspace toolbox

:)